stats/misc_user_stats.php

```diff
@@ -7,18 +7,14 @@
 require_login();
 
 $title = _("Miscellaneous User Statistics");
-$charts = [
-    ["id" => "average_hour_users_logging_on", "config" => average_hour_users_logging_on()],
-    ["id" => "users_by_language", "config" => users_by_language()],
-    ["id" => "users_by_country", "config" => users_by_country()],
-    ["id" => "new_users", "config" => new_users("month")],
+$graphs = [
+    ["barChart", "average_hour_users_logging_on", average_hour_users_logging_on()],
+    ["barChart", "users_by_language", users_by_language()],
+    ["barChart", "users_by_country", users_by_country()],
+    ["barChart", "new_users", new_users("month")],
 ];
 
-$js_data = '$(function(){';
-foreach ($charts as $chart) {
-    $js_data .= 'barChart("' . $chart["id"] . '", ' . json_encode($chart["config"]) . ');';
-}
-$js_data .= '});';
+$js_data = build_svg_graph_inits($graphs);
```

Comment:
I would put the `build_svg_graph_inits($graphs)` directly in the `output_header()` stanza and bypass creating the `$js_data` variable altogether like you've done elsewhere.